ts/features/wallet/components/WalletCardsContainer.tsx

```diff
@@ -66,6 +67,7 @@ const WalletCardsContainer = () => {
 
   return (
     <Animated.View style={{ flex: 1 }} layout={LinearTransition.duration(200)}>
+      <ItwOfflineWalletBanner />
```

Comment:
This can be placed directly into the `ItwWalletCardsContainer`, like we did here: ts/features/itwallet/wallet/components/ItwWalletCardsContainer.tsx

In this case we are sure the banner is only rendered when the wallet is active.